wireapp/wire-ios-mono
# feat: join an mls conference WPB-639

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-639" title="WPB-639" target="_blank"><img alt="Task" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10818?size=medium" />WPB-639</a>  [iOS] Join an MLS conference call
  </td></table>
  <br />
 






----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?
Added support for joining (answering) an MLS conference. Changes include:

- propagate errors when answering a call
- factor out setting up the mls conference from the start call method
- also set up the mls conference when answering a call

### Testing

#### Test Coverage

- tested manually
- unit test asserting answering an mls conference

#### How to Test
- Log in to MLS enabled backend (in this case Anta)
- Start an MLS group
- Start an MLS conference from a Chrome client
- Answer the MLS conference from the iOS client
- Assert that you hear and see the remote side

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
